### PR TITLE
Move front matter content to FAQs

### PR DESCRIPTION
**Pull request by Chilipp**
This PR covers the third point in https://github.com/RSE-leaders/SORSE20/issues/15#issuecomment-642940625, moving the current content of the landing page into an FAQ section.

If we merge this PR, we 

*  replace the current text on the landing page by the welcome text mentioned in https://github.com/RSE-leaders/SORSE20/issues/15#issuecomment-642934049
* create an [FAQ page](https://99-267395254-gh.circle-artifacts.com/0/SORSE20/faq/index.html) that is automatically populated and sorted by the tags of the corresponding FAQ 
* sort the former questions from the landing page into [About](https://99-267395254-gh.circle-artifacts.com/0/SORSE20/faq/index.html#about) and [How it works](https://99-267395254-gh.circle-artifacts.com/0/SORSE20/faq/index.html#how-it-works) sections

As mentioned in https://github.com/RSE-leaders/SORSE20/issues/15#issuecomment-641529419, I feel like the current questions rather read like an FAQ than an about. But I understand that an *About* section is one common way to get to know a web page. Therefore I also added the corresponding FAQ-section to the main nav. 

image

@TeriForey: Do you think this is okay?

### CODE REVIEW

**Comment by TeriForey** [comment]
It looks good but I'm not sure about the "share" buttons at the end of each item. It doesn't feel right for parts of an FAQ to be shared like events/news items. Could you remove those buttons, unless you have a strong opinion on keeping them?

File: _config.yml [hunk 1]
```diff
@@ -98,3 +98,18 @@ defaults:
...
       layout: single
       author_profile: false
+  # _faq
+  - scope:
+      path: ""
+      type: faq
+    values:
+      layout: single
+      author_profile: false
+      share: true
```
Comment (Chilipp) [inline _config.yml:108]:
```suggestion
      share: false
```
as suggested by @TeriForey in https://github.com/RSE-leaders/SORSE20/pull/64#issuecomment-643940335

**Comment by Chilipp** [comment]
> Could you remove those buttons, unless you have a strong opinion on keeping them?
good point @TeriForey, done with https://github.com/RSE-leaders/SORSE20/pull/64/commits/c2bee56249e64cb6e449b213debd9cb0780b439c (see https://104-267395254-gh.circle-artifacts.com/0/SORSE20/faq/howto/attendance/index.html for instance).

**Comment by TeriForey** [comment]
Perfect @Chilipp thank you!

**Pull request merged**